buildenv/jenkins/lineEndingsCheck

```diff
@@ -0,0 +1,96 @@
+/*******************************************************************************
+ * Copyright (c) 2017, 2017 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *******************************************************************************/
+
+Boolean FAIL = false
+String SRC_REPO = 'https://github.com/AdamBrousseau/openj9.git'
+String[] BAD_FILES
+String HASH = '#'
+String HASHES = '###################################'
+
+stage('Line Endings Check') {
+    node ('master') {
+        timestamps {
+            git url: SRC_REPO
+            sh "git fetch --tags --progress origin +refs/pull/*:refs/remotes/origin/pr/*"
+            sh "git checkout ${sha1}"
+            sh 'git fetch origin'
+            sh "git diff --diff-filter=ACM --name-only origin/${ghprbTargetBranch} HEAD"
+            FILES = sh (
+                script: "git diff --diff-filter=ACM --name-only origin/${ghprbTargetBranch} HEAD",
+                returnStdout: true
+            ).trim()
+            def FILES_LIST = FILES.split("\\r?\\n")
+            FILES_LIST.each() {
+                println "Checking file: '${it}'"
+                TYPE = sh (
+                    script: "file -b '${it}'",
+                    returnStdout: true
+                ).trim()
+
+                switch (TYPE) {
+                    case ~/empty/:
+                        echo "Empty file: '${it}'"
+                        break
+                    case ~/.*text.*/:
+                        switch (it) {
+                            case ~/.*\.bat|.*\.cmd/:
```

Comment:
The bash script I offered did a case-insensitive match for `*.bat` and `*.cmd` files. Does this do the same?

Reply:
Added .toLowerCase() on the switch of the filename